starfish/test/intensity_table/test_intensity_table_coords.py

```diff
@@ -89,3 +89,39 @@ def test_tranfering_physical_coords_to_intensity_table():
         physical_z = spot[Coordinates.Z.value].data
         assert np.isclose(physical_coords[PhysicalCoordinateTypes.Z_MAX],
                           (physical_z * 2) - physical_coords[PhysicalCoordinateTypes.Z_MIN])
+
+
+def test_tranfering_physical_coords_to_expression_matrix():
+    stack_shape = OrderedDict([(Axes.ROUND, 3), (Axes.CH, 2),
+                               (Axes.ZPLANE, 1), (Axes.Y, 50), (Axes.X, 40)])
+
+    physical_coords = OrderedDict([(PhysicalCoordinateTypes.X_MIN, 1),
+                                   (PhysicalCoordinateTypes.X_MAX, 2),
+                                   (PhysicalCoordinateTypes.Y_MIN, 4),
+                                   (PhysicalCoordinateTypes.Y_MAX, 6),
+                                   (PhysicalCoordinateTypes.Z_MIN, 1),
+                                   (PhysicalCoordinateTypes.Z_MAX, 3)])
+
+    stack = test_utils.imagestack_with_coords_factory(stack_shape, physical_coords)
+    codebook = test_utils.codebook_array_factory()
+
+    intensities = IntensityTable.synthetic_intensities(
+        codebook,
+        num_z=stack_shape[Axes.ZPLANE],
+        height=stack_shape[Axes.Y],
+        width=stack_shape[Axes.X],
+        n_spots=NUMBER_SPOTS
+    )
+
+    intensities = intensity_table_coordinates. \
+        transfer_physical_coords_from_imagestack_to_intensity_table(stack, intensities)
+
+    expression_matrix = intensities.to_expression_matrix()
+
+    # Assert that coords were transferred
+    xc = expression_matrix.coords[Coordinates.X]
+    yc = expression_matrix.coords[Coordinates.Y]
+    zc = expression_matrix.coords[Coordinates.Z]
+    assert xc.size == NUMBER_SPOTS
```

Comment:
this is puzzling to me.  this implies we have a cell for every spot?  what's the point of such a target assignment (basically none?)

Reply:
I thought it just implied that every spot had an xc,yc,zc coordinate?

Reply:
expression matrix is supposed to be a cells x gene plot.